tasks/filter-project-data/README.md

```diff
@@ -0,0 +1,52 @@
+# Filter Project Data task
+
+Concourse task to merge data for a particular project from [`targets.json`](../../config/targets.json) with the data from [the 18F Team API](https://team-api.18f.gov/public/api/).
+
+## Configuring projects
+
+Any keys in [`targets.json`](../../config/targets.json) will override the corresponding values in the input (`project-data/project.json`). For example, having the following in `project.json`:
+
+```javascript
+{
+  "name": "someproject",
+  "otherfield": "something"
+  // ...
+}
+```
+
+and the following in `targets.json`:
+
+```javascript
+[
+  {
+    "name": "someproject",
+    "links": [
+      "https://staging.someproject.com"
+    ]
+  },
+  // ...
+]
+```
+
+will result in an output of
+
+```javascript
+{
+  "name": "someproject",
+  "links": [
+    "https://staging.someproject.com"
+  ],
+  "otherfield": "something"
+  // ...
+}
```

Comment:
This example just looks like a merge, not an overwrite. To emphasize the overwrite, care to add a `links` section to `project.json`?